### PR TITLE
Sync UI

### PR DESCRIPTION
**Pull request by yswdqz**
- [ ] If this pull request closes/resolves/fixes an existing issue, replace the issue number. Closes #<issue number>.
- [x] Update the [`CHANGES` log](https://github.com/apache/skywalking/blob/master/docs/en/changes/changes.md).

### CODE REVIEW

File: docs/en/changes/changes.md [hunk 1]
```diff
@@ -40,6 +40,9 @@
 * Fix filter ID when ReadRecords metric associates with trace.
 * Add AWS API Gateway menu.
 * Change trace profiling protocol.
+* polish tooltips for trace profiling widget.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:43]: We don't need this in the changelog. This bug was never released.

**Review by wu-sheng — COMMENTED** [review]
You could choose including the sync in your feature PR too.

**Comment by yswdqz** [comment]
Got it.

File: docs/en/changes/changes.md [hunk 2]
```diff
@@ -40,6 +40,8 @@
 * Fix filter ID when ReadRecords metric associates with trace.
 * Add AWS API Gateway menu.
 * Change trace profiling protocol.
+* fix:incorrect node name length calculation.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:43]: This one should not be in the change log too. Never released.

Comment (wu-sheng) [inline docs/en/changes/changes.md:43]:
```suggestion
```

File: docs/en/changes/changes.md [hunk 3]
```diff
@@ -40,6 +40,7 @@
 * Fix filter ID when ReadRecords metric associates with trace.
 * Add AWS API Gateway menu.
 * Change trace profiling protocol.
+* add Redis menu.
```
Comment (wu-sheng) [inline docs/en/changes/changes.md:43]:
```suggestion
* Add Redis menu.
```

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**